.env.development

```diff
@@ -10,4 +10,4 @@ REACT_APP_ALCHEMY_API_KEY=<ALCHEMY_API_KEY>
 REACT_APP_REGISTRY_CONTRACT="0xCFa36F3692b19FF9472aEc18f7dcf5EB0A29A633"
 REACT_APP_SCANNER_URL=https://goerli.etherscan.io/tx
 REACT_APP_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/horse-link/hl-protocol-goerli
-REACT_APP_MARKET_ORACLE_CONTRACT="0x5b559E0E44aeB2bccfd82CACF9d877EbDd28F116"
+REACT_APP_MARKET_ORACLE_CONTRACT="0x5b559E0E44aeB2bccfd82CACF9d877EbDd28F116"
```

Comment:
the new REACT_APP_MARKET_ORACLE_CONTRACT is 0x0e7983965d231b98804AFbEF134b73c6CF56e8Ec

Reply:
nvm, lets swap that over in its own PR